demo-kubelet.md

```diff
@@ -1,22 +1,22 @@
 # Akri and Krustlet integration demo
```

Comment:
this is the same title as the `demo-krustlet.md` document. Try to distinguish the difference in the titles